Signal/src/ViewControllers/GifPicker/GifPickerViewController.swift

```diff
@@ -365,15 +367,15 @@ class GifPickerViewController: OWSViewController, UISearchBarDelegate, UICollect
                 return
             }
 
-            strongSelf.tryToSearch()
+            strongSelf.tryToSearch(dismissKeyboard: false)
```

Comment:
I think we only want to dismiss when the user explicitly taps the "search" button right? 

Currently we'll also dismiss upon retry right?

It seems like the dismiss logic should live with the button press in `searchBarSearchButtonClicked`

```
    public func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
        tryToSearch()
        self.searchBar.resignFirstResponder()
    }
```

No biggie though.

Reply:
Yup, that's better. Done.